### PR TITLE
fix: jumpy nav on safari mweb

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-2407?atlOrigin=eyJpIjoiNGFiZDRlOWMxNzRlNDVmYzlmMmQwYTllY2JmMmFjMTIiLCJwIjoiaiJ9

note: I opened the vercel build link on my mobile device and tested in safari. you can't see the bug in the safari desktop mobile developer testing env.

### CODE REVIEW

**Comment by JFrankfurt** [comment]
on mobile safari the nav bar is now stuck in the middle of the screen for me when I scroll down 😬 

image

**Review by JFrankfurt — CHANGES_REQUESTED** [review]
see comment

File: src/pages/Pool/index.tsx [hunk 1]
```diff
@@ -165,7 +165,7 @@ function WrongNetworkCard() {
   const theme = useTheme()
 
   return (
-    <>
+    <div style={{ height: '100vh' }}>
       <PageWrapper>
```
Comment (JFrankfurt) [inline src/pages/Pool/index.tsx:169]: it looks like `PageWrapper` already has the 100vh style applied--why is the wrapper necessary here?

File: src/pages/Pool/index.tsx [hunk 2]
```diff
@@ -263,7 +263,7 @@ export default function Pool() {
 
   return (
     <Trace page={PageName.POOL_PAGE} shouldLogImpression>
-      <>
+      <div style={{ height: '100vh' }}>
```
Comment (JFrankfurt) [inline src/pages/Pool/index.tsx:266]: same q re: PageWrapper having 100vh applied already

File: src/pages/Pool/index.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/pages/Pool/index.tsx:169]: the PageWrapper is not the the outermost div, this extra div is necessary to include SwitchLocaleLink. Putting switchLocaleLink inside PageWrapper has other undesired effects on the Pool component styling

**Comment by lynnshaoyu** [comment]
> on mobile safari the nav bar now sticks in place for me when I scroll down 😬
> 
> image

oof thanks for catching that last commit did NOT do what I thought it was gonna😞

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**